package.json

```diff
@@ -4,6 +4,7 @@
   "license": "GPL-3.0-only",
   "author": "Piotr Roslaniec <p.roslaniec@gmail.com>",
   "scripts": {
+    "postinstall": "pnpm build || true && echo 'Ignoring build errors on setup until we publish the first version'",
```

Comment:
We should track this

Reply:
Opened #307

Reply:
Tracked here: https://github.com/nucypher/nucypher-ts/issues/308

Reply:
😆